integrations-and-supported-tools/sklearn/notebooks/Neptune_Scikit_learn.ipynb

```diff
@@ -1027,7 +1031,9 @@
    },
    "outputs": [],
    "source": [
-    "run[\"confusion-matrix\"] = npt_utils.create_confusion_matrix_chart(rfc, X_train, X_test, y_train, y_test)"
+    "run[\"confusion-matrix\"] = npt_utils.create_confusion_matrix_chart(\n",
+    "    rfc, X_train, X_test, y_train, y_test\n",
```

Comment:
Can you strip this extra space?

Reply:
this is indentation
![image](https://user-images.githubusercontent.com/41324509/210603042-5775cd28-91c3-4e93-bc17-c0f870793126.png)